src/app/auth/components/authorisation.component.ts

```diff
@@ -0,0 +1,48 @@
+import { Component, ElementRef, ViewChild } from '@angular/core';
+import { FormGroup, FormBuilder, FormControl, Validators } from '@angular/forms';
+
+
+@Component({
+  selector:"authorisation",
+  templateUrl: "authorisation.component.html",
+  styleUrls: ["authorisation.component.scss"]
+})
+
+export class AuthorisationComponent {
+
+  public greenEye: boolean;
```

Comment:
I also have green eyes, but unfortunately I can't understand what doing this variable. I think what would be much better if we can rename it to "showPassword" for example